site/src/components/dismiss.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Dismiss
+description: The dismiss component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - dismiss
+---
+
+# Dismiss
+
+Dismiss component can be used within any other component that can be removed from the page. For example, modal, notification, banner, etc.
+
+```html
+<button type="button" class="dismiss" data-dismiss data-bi-name="close">
+	<span class="visually-hidden">Dismiss</span>
+	<span class="icon" aria-hidden="true">
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448" class="fill-current-color">
+			<path
+				d="M269.254 224l137.373 137.373-45.254 45.254L224 269.254 86.627 406.627l-45.255-45.254L178.746 224 41.373 86.627l45.255-45.255L224 178.746 361.373 41.373l45.254 45.255L269.254 224z"
+			/>
+		</svg>
+	</span>
+</button>
+```
+
+## Usage
+
+There are a few things to keep in mind when using dismiss component:
+
+- dismiss component has to have the `data-dismiss` attribute
+- the component it is used within has to have `data-dismissable` attribute
+
+Below is an example usage with `notification` component.
+
+```html
+<div class="notification" id="example-notification-01" data-dismissable data-testid="notification">
```

Comment:
`data-test-id`?

Reply:
Do we even want to have `data-test-id` attributes at all in our Atlas docs? (I vote we remove it)

Reply:
`data-testid` is used in the integration test.

Also, it is `data-testid` instead of `data-test-id` cause I read in the playwright documentation that's the default attribute name for targeting elements by id:

> By default, the data-testid attribute is used as a test id

https://playwright.dev/docs/api/class-locator#locator-get-by-test-id

Reply:
Hey that is cool! I am starting to think that @BenDMyers is correct though. Maybe our use cases are simple enough that we don't require a specific testId selector in Atlas. The unfortunate thing is that they clutter the code examples.

If we really want to keep the ids (my vote is to remove), let's keep them the same as our other repos, and use in our playwright config.

```
use: {
  testIdAttribute: 'data-test-id'
}
```

Reply:
Updated test to not use playwright's id at all, so examples look clean.